cypress_test/integration_tests/common/impress_helper.js

```diff
@@ -119,21 +119,19 @@ function removeShapeSelection() {
 	cy.log('>> removeShapeSelection - start');
 
 	// Remove selection with clicking on the top-left corner of the slide
-	cy.waitUntil(function() {
-		cy.cGet('.leaflet-canvas-container canvas')
-			.then(function(items) {
-				var XPos = items[0].getBoundingClientRect().left + 10;
-				var YPos = items[0].getBoundingClientRect().top + 10;
-				cy.cGet('body').click(XPos, YPos);
-				cy.cGet('body').type('{esc}');
-				cy.cGet('body').type('{esc}');
-			});
+	cy.cGet('.leaflet-canvas-container canvas')
```

Comment:
looks sensible to not repeat it, we can get delayed response and incorrectly act on that